prover/zkevm/checker.go

```diff
@@ -32,6 +32,7 @@ func CheckerZkEvm(tl *config.TracesLimits) *ZkEvm {
 			Arithmetization: arithmetization.Settings{
 				Limits:            tl,
 				OptimisationLevel: &mir.DEFAULT_OPTIMISATION_LEVEL,
+				RelaxedMode:       false,
```

Comment:
It would be great if we could rename `RelaxedMode` to something more intuitive, such as `SkipCompatibilityCheck`, `SkipConstraintVersionCheck`, `DisableCompatibilityCheck`, or `DisableConstraintVersionCheck`.

This would help others without prior context better understand the purpose of this configuration.
Especially if we decide to provide this as a CLI flag for the full or partial prover, a clearer name would make it easier for the Ops team and release managers to understand its function.